samza-core/src/main/java/org/apache/samza/runtime/LocalApplicationRunner.java

```diff
@@ -59,7 +60,7 @@
 
   private final String uid;
   private final Set<StreamProcessor> processors = ConcurrentHashMap.newKeySet();
-  private final CountDownLatch shutdownLatch = new CountDownLatch(1);
+  /* package private */ final CountDownLatch shutdownLatch = new CountDownLatch(1);
```

Comment:
please use a package private method to retrieve it.

Reply:
done